docs/cel2/l2-operator-guide.md

```diff
@@ -29,47 +29,43 @@ The instructions below are not yet fully usable, as some values need to be set.
+TODO: Stay tuned for Alfajores migration block number
 :::
 
 This will automatically prevent your node from processing blocks higher than `l2migrationblock - 1`.
 
 ## Starting an L2 node
 
-### Overview
-
-Three components are required to run an L2 node:
-
-- [eigenda-proxy](https://github.com/Layr-Labs/eigenda-proxy/tree/main): proxy interface between EigenDA and the OP stack
-- op-node: the consensus layer
-- op-geth: the execution layer
-
-For op-geth, there are three options to get started, ranked by ease and the level of trust required:
+Node operators who wish to run an L2 node have three options, ranked by ease and the level of trust required:
 
 1. Start an L2 node with snap sync. This option does not require running the migration script.
 2. Start an L2 node with the provided L1 chaindata.
 3. Migrate the L1 chaindata manually.
 
-Using snap sync offers a simpler and faster experience, but it cannot be used if you want to run an archive node. In that case, you will need to use an archive node snapshot or migrate your own archive data from an L1 node.
+Snap sync provides a simpler and faster setup experience. However, it is not suitable if you plan to run an archive node. In that case, you'll need to either use an archive node snapshot or migrate your own archive data from an L1 node. While both options follow a similar process, there are some differences, particularly in how you prepare the chaindata from the L1. Nonetheless, the majority of the service configuration remains the same across all options.
 
 :::info
 If you plan to migrate your own chaindata, we recommend getting familiar with the instructions and running the pre-migration 1-3 days ahead of the full migration.
 :::
 
-The next sections will provide more information on running each component, including the different options for op-geth.
+To set up your node, you'll need to run three services: `op-node`, `op-geth`, and `eigenda-proxy`. op-node serves as the consensus client for the L2 node, op-geth as the execution client, and eigenda-proxy acts as the interface between your L2 node and the data availability layer [EigenDA](https://www.blog.eigenlayer.xyz/intro-to-eigenda-hyperscale-data-availability-for-rollups/).
+
+If you plan to run multiple L2 nodes, you’ll need separate instances of `op-node` and `op-geth` for each node, but you can share a single instance of `eigenda-proxy`. Additionally, you'll need an endpoint for the L1 RPC, which can either be a public node or your own L1 node. For Alfajores, the L1 used is [Holesky](https://chainlist.org/chain/17000).
+
+- [Running EigenDA Proxy](#running-eigenda-proxy)
+- [Running op-node](#running-op-node)
+- [Running op-geth](#running-op-geth)
 
 ### Running EigenDA Proxy
 
-:::info
 These are brief instructions for running an eigenda-proxy instance. For more detailed instructions, please refer to the [repository README](https://github.com/Layr-Labs/eigenda-proxy/tree/main?tab=readme-ov-file#deployment-guide).
```

Comment:
I'd keep this piece of info symmetric with the reference to the op-geth documentation around line 183, either both as an info callout, or both in plaintext.